jobs/pr_gate/test_config/FIT/config.ini

```diff
@@ -0,0 +1,33 @@
+[DEFAULT]
```

Comment:
This file is not used in FIT.  We currently use cit_defaults.json, which also, is getting deprecated when we remove CIT.